src/ontology/mondo-edit.obo

```diff
@@ -438012,13 +438013,18 @@ xref: GARD:21824 {source="MONDO:GARD"}
 xref: ICD10CM:I63.5 {source="Orphanet:439175", source="Orphanet:439175/ntbt"}
 xref: MEDGEN:1803840 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:439175 {source="MONDO:equivalentTo"}
+xref: SCTID:1197363004 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680049 {source="MEDGEN:1803840", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
+is_a: MONDO:0000473 {source="Orphanet:439175"} ! arterial disorder
+is_a: MONDO:0005098 {source="Orphanet:439175"} ! stroke disorder
+is_a: MONDO:0005299 {source="Orphanet:439175"} ! brain ischemia
```

Comment:
@MeeSiing these "orphanet sources" need to be added to [this document](https://docs.google.com/spreadsheets/d/1YnJEAgwkGksQyZ86JSq5cS3gKvEmF9-5kbHzs9L-dE8/edit?gid=0#gid=0) to indicate that orphanet did not say that these are the parents (which I assumed based on [here](https://www.ebi.ac.uk/ols4/ontologies/ordo/classes/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_439175)), but information in orphanet supported these SC. Otherwise, these sources will be removed through our automated pipelines.